### DISCUSSION TITLE
The alert "KubeletPodStartUpLatencyHigh" is not getting triggered in Prometheus

### DISCUSSION THREAD

**Discussion by edwin24mvrks** [prometheus-operator/prometheus-operator#6054] [General]
Issue: 
The alert "KubeletPodStartUpLatencyHigh" is not getting triggered in Prometheus
 
Environment: 
SLES OS version: 15-SP4, kubernetes version: 1.26.4, Prometheus version: v2.42.0
 
How did I try to trigger this alert?
I tried to give extreme stress on Kubelet worker in terms of resource usage, but the metrics is not going above 10, and this alert gets triggered only if this value goes > 60. 
 
Help/support needed: 
How do I successfully trigger this alert?

**Comment by slashpai** [comment]
I am moving this to discussion as this alert is part of [kube-prometheus alert](https://github.com/prometheus-operator/kube-prometheus/blob/b9d1ff5a8848bcedb465d60dd61775debb881534/manifests/kubernetesControlPlane-prometheusRule.yaml#L659-L668) and not prometheus-operator question

**Comment by slashpai** [comment]
So looking at the alert query IIUC it calculates the 99th percentile of kubelet_pod_worker_duration_seconds_bucket and then filter for instances where the kubelet_node_name metric is greater than 60.

May be you can simulate the condition by inflating pod startup time?
